tests/acquia_lift.test

```diff
@@ -0,0 +1,561 @@
+ * Tests for Acquia Lift Profiles module.
+ */
+
+/**
+ * Tests Acquia Lift functionality.
+ */
+class AcquiaLiftsWebTest extends DrupalWebTestCase {
+
+  protected $admin_user;
+  protected $admin_button_text;
+
+  public static function getInfo() {
+    return array(
+      'name' => t('Acquia Lift Tests'),
+      'description' => t('Tests basic functionality of Acquia Lift module.'),
+      'group' => t('Acquia'),
+    );
+  }
+
+  function setUp() {
+    parent::setUp(array('acquia_lift'));
+    $this->admin_button_text = t('Save configuration');
+    $this->admin_user = $this->drupalCreateUser(array(
+      'access administration pages',
+      'administer acquia_lift configuration',
+      'administer content types',
+      'administer nodes',
+      'bypass node access'
+    ));
+
+    // Set up a variety of nodes with different term combinations.
+    $this->tags['one_tag'] = array($this->randomName());
+    $this->tags['two_tags'] = array($this->randomName(), $this->randomName());
+    $this->articles['one_tag'] =
+      $this->createArticleWithTerms($this->tags['one_tag']);
+    $this->articles['two_tags'] =
+      $this->createArticleWithTerms($this->tags['two_tags']);
+  }
+
+  function tearDown() {
+    $vars = array(
+      'acquia_lift_identity_param',
+      'acquia_lift_identity_type_param',
+      'acquia_lift_default_identity_type',
+      'acquia_lift_field_mappings',
+      'acquia_lift_udf_mappings',
+      'acquia_lift_identity_param',
+      'acquia_lift_identity_type_param',
+      'acquia_lift_account_id',
+      'acquia_lift_site_id',
+      'acquia_lift_assets_url',
+      'acquia_lift_decision_api_url',
+      'acquia_lift_oauth_url',
+      'acquia_lift_ignore_path_patterns',
+      'acquia_lift_content_replacement_mode',
+    );
+    foreach ($vars as $index => $name) {
+      variable_del($name);
+    }
+    parent::tearDown();
+  }
+
+  /**
+   * Tests the configuration form for Acquia Lift.
+   */
+  function testAdminSettingsForm() {
+    $this->drupalLogin($this->admin_user);
+    $this->drupalGet('/admin/config/services/acquia_lift');
+
+    $submit_confirm_text = t('The configuration settings have been saved.');
+    $submit_button_text = $this->admin_button_text;
+
+    // Confirm expected fields
+    $fields = array(
+      'acquia_lift_account_id',
+      'acquia_lift_site_id',
+      'acquia_lift_assets_url',
+      'acquia_lift_decision_api_url',
+      'acquia_lift_oauth_url',
+      'acquia_lift_identity_param',
+      'acquia_lift_identity_type_param',
+      'acquia_lift_default_identity_type',
+      'acquia_lift_ignore_path_patterns',
+      'acquia_lift_content_replacement_mode',
+      'acquia_lift_field_mappings[content_section]',
+      'acquia_lift_field_mappings[content_keywords]',
+      'acquia_lift_field_mappings[persona]',
+    );
+    for ($i = 1; $i <= 50; $i++) {
+      $fields[] = 'acquia_lift_udf_mappings[person][person_udf' . $i . ']';
+    }
+    for ($i = 1; $i <= 20; $i++) {
```

Comment:
Same as above - just want to double-check as the original/current d7 code has it at 20.  If this is true we need to update lift 2 also.